Tests don't compile with `--all-features`

```
$ cargo check --tests --all-features
    Checking nimiq-network-libp2p v0.1.0 (core-rs-albatross/network-libp2p)
error[E0433]: failed to resolve: use of undeclared type `OffsetTime`
   --> network-libp2p/tests/request_response.rs:194:22
    |
194 |             Arc::new(OffsetTime::new()),
    |                      ^^^^^^^^^^ use of undeclared type `OffsetTime`
    |
help: consider importing this struct
    |
1   + use nimiq_utils::time::OffsetTime;
    |

error[E0433]: failed to resolve: use of undeclared type `OffsetTime`
   --> network-libp2p/tests/request_response.rs:204:22
    |
204 |             Arc::new(OffsetTime::new()),
    |                      ^^^^^^^^^^ use of undeclared type `OffsetTime`
    |
help: consider importing this struct
    |
1   + use nimiq_utils::time::OffsetTime;
    |

error[E0433]: failed to resolve: use of undeclared type `OffsetTime`
   --> network-libp2p/tests/request_response.rs:214:22
    |
214 |             Arc::new(OffsetTime::new()),
    |                      ^^^^^^^^^^ use of undeclared type `OffsetTime`
    |
help: consider importing this struct
    |
1   + use nimiq_utils::time::OffsetTime;
    |

error[E0433]: failed to resolve: use of undeclared type `OffsetTime`
   --> network-libp2p/tests/request_response.rs:224:22
    |
224 |             Arc::new(OffsetTime::new()),
    |                      ^^^^^^^^^^ use of undeclared type `OffsetTime`
    |
help: consider importing this struct
    |
1   + use nimiq_utils::time::OffsetTime;
    |

error[E0277]: the trait bound `nimiq_network_libp2p::Config: Clone` is not satisfied
   --> network-libp2p/tests/request_response.rs:195:13
    |
193 |         let net1 = Network::new(
    |                    ------------ required by a bound introduced by this call
194 |             Arc::new(OffsetTime::new()),
195 |             network_config(addr1.clone()),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Clone` is not implemented for `nimiq_network_libp2p::Config`
    |
note: required by a bound in `nimiq_network_libp2p::Network::new`
   --> core-rs-albatross/network-libp2p/src/network.rs:235:75
    |
235 | ...skExecutor + Send + Clone + 'static) -> Self {
    |                        ^^^^^ required by this bound in `Network::new`

error[E0277]: expected a `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` closure, found `nimiq_network_libp2p::Config`
   --> network-libp2p/tests/request_response.rs:195:13
    |
193 |         let net1 = Network::new(
    |                    ------------ required by a bound introduced by this call
194 |             Arc::new(OffsetTime::new()),
195 |             network_config(addr1.clone()),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected an `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` closure, found `nimiq_network_libp2p::Config`
    |
    = help: the trait `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` is not implemented for `nimiq_network_libp2p::Config`
    = note: required for `nimiq_network_libp2p::Config` to implement `nimiq_primitives::task_executor::TaskExecutor`
note: required by a bound in `nimiq_network_libp2p::Network::new`
   --> core-rs-albatross/network-libp2p/src/network.rs:235:53
    |
235 | ..., executor: impl TaskExecutor + Send + Clone + 'static) -> Self {
    |                     ^^^^^^^^^^^^ required by this bound in `Network::new`

error[E0061]: this function takes 2 arguments but 3 arguments were supplied
   --> network-libp2p/tests/request_response.rs:193:20
    |
193 |            let net1 = Network::new(
    |                       ^^^^^^^^^^^^
194 |                Arc::new(OffsetTime::new()),
195 |                network_config(addr1.clone()),
    |  ___________________________________________-
196 | |/             Box::new(|fut| {
197 | ||                 tokio::spawn(fut);
198 | ||             }),
    | ||              -
    | ||______________|
    |  |______________help: remove the extra argument
    |                 unexpected argument of type `Box<{closure@network-libp2p/tests/request_response.rs:196:22: 196:27}>`
    |
note: associated function defined here
   --> core-rs-albatross/network-libp2p/src/network.rs:235:18
    |
235 |     pub async fn new(config: Config, executor: impl TaskExecutor + Send + Clone + 'static)...
    |                  ^^^

error[E0277]: the trait bound `nimiq_network_libp2p::Config: Clone` is not satisfied
   --> network-libp2p/tests/request_response.rs:205:13
    |
203 |         let net2 = Network::new(
    |                    ------------ required by a bound introduced by this call
204 |             Arc::new(OffsetTime::new()),
205 |             network_config(addr2.clone()),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Clone` is not implemented for `nimiq_network_libp2p::Config`
    |
note: required by a bound in `nimiq_network_libp2p::Network::new`
   --> core-rs-albatross/network-libp2p/src/network.rs:235:75
    |
235 | ...skExecutor + Send + Clone + 'static) -> Self {
    |                        ^^^^^ required by this bound in `Network::new`

error[E0277]: expected a `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` closure, found `nimiq_network_libp2p::Config`
   --> network-libp2p/tests/request_response.rs:205:13
    |
203 |         let net2 = Network::new(
    |                    ------------ required by a bound introduced by this call
204 |             Arc::new(OffsetTime::new()),
205 |             network_config(addr2.clone()),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected an `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` closure, found `nimiq_network_libp2p::Config`
    |
    = help: the trait `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` is not implemented for `nimiq_network_libp2p::Config`
    = note: required for `nimiq_network_libp2p::Config` to implement `nimiq_primitives::task_executor::TaskExecutor`
note: required by a bound in `nimiq_network_libp2p::Network::new`
   --> core-rs-albatross/network-libp2p/src/network.rs:235:53
    |
235 | ..., executor: impl TaskExecutor + Send + Clone + 'static) -> Self {
    |                     ^^^^^^^^^^^^ required by this bound in `Network::new`

error[E0061]: this function takes 2 arguments but 3 arguments were supplied
   --> network-libp2p/tests/request_response.rs:203:20
    |
203 |            let net2 = Network::new(
    |                       ^^^^^^^^^^^^
204 |                Arc::new(OffsetTime::new()),
205 |                network_config(addr2.clone()),
    |  ___________________________________________-
206 | |/             Box::new(|fut| {
207 | ||                 tokio::spawn(fut);
208 | ||             }),
    | ||              -
    | ||______________|
    |  |______________help: remove the extra argument
    |                 unexpected argument of type `Box<{closure@network-libp2p/tests/request_response.rs:206:22: 206:27}>`
    |
note: associated function defined here
   --> core-rs-albatross/network-libp2p/src/network.rs:235:18
    |
235 |     pub async fn new(config: Config, executor: impl TaskExecutor + Send + Clone + 'static)...
    |                  ^^^

error[E0277]: the trait bound `nimiq_network_libp2p::Config: Clone` is not satisfied
   --> network-libp2p/tests/request_response.rs:215:13
    |
213 |         let net3 = Network::new(
    |                    ------------ required by a bound introduced by this call
214 |             Arc::new(OffsetTime::new()),
215 |             network_config(addr3.clone()),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Clone` is not implemented for `nimiq_network_libp2p::Config`
    |
note: required by a bound in `nimiq_network_libp2p::Network::new`
   --> core-rs-albatross/network-libp2p/src/network.rs:235:75
    |
235 | ...skExecutor + Send + Clone + 'static) -> Self {
    |                        ^^^^^ required by this bound in `Network::new`

error[E0277]: expected a `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` closure, found `nimiq_network_libp2p::Config`
   --> network-libp2p/tests/request_response.rs:215:13
    |
213 |         let net3 = Network::new(
    |                    ------------ required by a bound introduced by this call
214 |             Arc::new(OffsetTime::new()),
215 |             network_config(addr3.clone()),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected an `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` closure, found `nimiq_network_libp2p::Config`
    |
    = help: the trait `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` is not implemented for `nimiq_network_libp2p::Config`
    = note: required for `nimiq_network_libp2p::Config` to implement `nimiq_primitives::task_executor::TaskExecutor`
note: required by a bound in `nimiq_network_libp2p::Network::new`
   --> core-rs-albatross/network-libp2p/src/network.rs:235:53
    |
235 | ..., executor: impl TaskExecutor + Send + Clone + 'static) -> Self {
    |                     ^^^^^^^^^^^^ required by this bound in `Network::new`

error[E0061]: this function takes 2 arguments but 3 arguments were supplied
   --> network-libp2p/tests/request_response.rs:213:20
    |
213 |            let net3 = Network::new(
    |                       ^^^^^^^^^^^^
214 |                Arc::new(OffsetTime::new()),
215 |                network_config(addr3.clone()),
    |  ___________________________________________-
216 | |/             Box::new(|fut| {
217 | ||                 tokio::spawn(fut);
218 | ||             }),
    | ||              -
    | ||______________|
    |  |______________help: remove the extra argument
    |                 unexpected argument of type `Box<{closure@network-libp2p/tests/request_response.rs:216:22: 216:27}>`
    |
note: associated function defined here
   --> core-rs-albatross/network-libp2p/src/network.rs:235:18
    |
235 |     pub async fn new(config: Config, executor: impl TaskExecutor + Send + Clone + 'static)...
    |                  ^^^

error[E0277]: the trait bound `nimiq_network_libp2p::Config: Clone` is not satisfied
   --> network-libp2p/tests/request_response.rs:225:13
    |
223 |         let net4 = Network::new(
    |                    ------------ required by a bound introduced by this call
224 |             Arc::new(OffsetTime::new()),
225 |             network_config(addr4.clone()),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Clone` is not implemented for `nimiq_network_libp2p::Config`
    |
note: required by a bound in `nimiq_network_libp2p::Network::new`
   --> core-rs-albatross/network-libp2p/src/network.rs:235:75
    |
235 | ...skExecutor + Send + Clone + 'static) -> Self {
    |                        ^^^^^ required by this bound in `Network::new`

error[E0277]: expected a `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` closure, found `nimiq_network_libp2p::Config`
   --> network-libp2p/tests/request_response.rs:225:13
    |
223 |         let net4 = Network::new(
    |                    ------------ required by a bound introduced by this call
224 |             Arc::new(OffsetTime::new()),
225 |             network_config(addr4.clone()),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected an `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` closure, found `nimiq_network_libp2p::Config`
    |
    = help: the trait `Fn<(Pin<Box<(dyn futures_util::Future<Output = ()> + std::marker::Send + 'static)>>,)>` is not implemented for `nimiq_network_libp2p::Config`
    = note: required for `nimiq_network_libp2p::Config` to implement `nimiq_primitives::task_executor::TaskExecutor`
note: required by a bound in `nimiq_network_libp2p::Network::new`
   --> core-rs-albatross/network-libp2p/src/network.rs:235:53
    |
235 | ..., executor: impl TaskExecutor + Send + Clone + 'static) -> Self {
    |                     ^^^^^^^^^^^^ required by this bound in `Network::new`

error[E0061]: this function takes 2 arguments but 3 arguments were supplied
   --> network-libp2p/tests/request_response.rs:223:20
    |
223 |            let net4 = Network::new(
    |                       ^^^^^^^^^^^^
224 |                Arc::new(OffsetTime::new()),
225 |                network_config(addr4.clone()),
    |  ___________________________________________-
226 | |/             Box::new(|fut| {
227 | ||                 tokio::spawn(fut);
228 | ||             }),
    | ||              -
    | ||______________|
    |  |______________help: remove the extra argument
    |                 unexpected argument of type `Box<{closure@network-libp2p/tests/request_response.rs:226:22: 226:27}>`
    |
note: associated function defined here
   --> core-rs-albatross/network-libp2p/src/network.rs:235:18
    |
235 |     pub async fn new(config: Config, executor: impl TaskExecutor + Send + Clone + 'static)...
    |                  ^^^

Some errors have detailed explanations: E0061, E0277, E0433.
For more information about an error, try `rustc --explain E0061`.
error: could not compile `nimiq-network-libp2p` (test "request_response") due to 16 previous errors
warning: build failed, waiting for other jobs to finish...
```

--- a/network-libp2p/Cargo.toml
+++ b/network-libp2p/Cargo.toml
@@ -86,11 +86,12 @@ libp2p = { version = "0.53.2", default-features = false, features = [
 
 [dev-dependencies]
 # In dev/testing we require more tokio features
-tokio = { version = "1.35", features = ["macros", "rt", "rt-multi-thread", "test-util", "tracing"] }
+tokio = { version = "1.35", features = ["macros", "rt", "rt-multi-thread", "test-util", "time", "tracing"] }
 
 nimiq-test-log = { workspace = true }
 
 [features]
+default = ["tokio-time"]
 metrics = ["prometheus-client"]
 tokio-time = ["tokio/time"]
 tokio-websocket = ["libp2p/dns", "libp2p/tcp", "libp2p/tokio", "libp2p/websocket"]

--- a/network-libp2p/tests/request_response.rs
+++ b/network-libp2p/tests/request_response.rs
@@ -191,7 +191,6 @@ impl TestNetwork {
         let addr4 = multiaddr![Memory(rng.gen::<u64>())];
 
         let net1 = Network::new(
-            Arc::new(OffsetTime::new()),
             network_config(addr1.clone()),
             Box::new(|fut| {
                 tokio::spawn(fut);
@@ -201,7 +200,6 @@ impl TestNetwork {
         net1.listen_on(vec![addr1.clone()]).await;
 
         let net2 = Network::new(
-            Arc::new(OffsetTime::new()),
             network_config(addr2.clone()),
             Box::new(|fut| {
                 tokio::spawn(fut);
@@ -211,7 +209,6 @@ impl TestNetwork {
         net2.listen_on(vec![addr2.clone()]).await;
 
         let net3 = Network::new(
-            Arc::new(OffsetTime::new()),
             network_config(addr3.clone()),
             Box::new(|fut| {
                 tokio::spawn(fut);
@@ -221,7 +218,6 @@ impl TestNetwork {
         net3.listen_on(vec![addr3.clone()]).await;
 
         let net4 = Network::new(
-            Arc::new(OffsetTime::new()),
             network_config(addr4.clone()),
             Box::new(|fut| {
                 tokio::spawn(fut);

